deps/xxhash/.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: c
```

Comment:
Remove this file, we're not using travis

Reply:
This is part of the xxhash repository. I'd like to keep our changes here to the bare minimum.